ios/FluentUI/Tooltip/Tooltip.swift

```diff
@@ -58,26 +61,26 @@ open class Tooltip: NSObject, TokenizedControlInternal {
             return
         }
 
-        let rootViewController = window.rootViewController
-        let rootView = rootViewController?.view
-        rootViewController?.addChild(tooltipViewController)
+        let hostVC = hostViewController ?? window.rootViewController
+        let hostView = hostVC?.view
```

Comment:
Should we be doing anything if the hostVC is nil? If no, maybe we could change it to a `guard let`?